README.md

```diff
@@ -16,48 +17,72 @@ All Saleor services started from a single repository
 $ git clone git@github.com:mirumee/saleor-platform.git --recursive --jobs 3
```

Comment:
This command does not clone submodules for you? If not, what git version you are using? What operating system?

Result on my machine:
```
❯ git clone git@github.com:mirumee/saleor-platform.git clonetest --recursive --jobs 3
Cloning into 'clonetest'...
remote: Enumerating objects: 9, done.
remote: Counting objects: 100% (9/9), done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 163 (delta 3), reused 4 (delta 1), pack-reused 154
Receiving objects: 100% (163/163), 58.62 KiB | 659.00 KiB/s, done.
Resolving deltas: 100% (78/78), done.
Submodule 'saleor' (git@github.com:mirumee/saleor.git) registered for path 'saleor'
Submodule 'saleor-dashboard' (git@github.com:mirumee/saleor-dashboard.git) registered for path 'saleor-dashboard'
Submodule 'saleor-storefront' (git@github.com:mirumee/saleor-storefront.git) registered for path 'saleor-storefront'
Cloning into '/Users/krzyh/code/clonetest/saleor'...
remote: Enumerating objects: 640, done.        
remote: Counting objects: 100% (640/640), done.        
remote: Compressing objects: 100% (258/258), done.        
remote: Total 173687 (delta 460), reused 481 (delta 382), pack-reused 173047        
Receiving objects: 100% (173687/173687), 116.23 MiB | 5.29 MiB/s, done.
Resolving deltas: 100% (126120/126120), done.
Cloning into '/Users/krzyh/code/clonetest/saleor-dashboard'...
remote: Enumerating objects: 495, done.        
remote: Counting objects: 100% (495/495), done.        
remote: Compressing objects: 100% (296/296), done.        
remote: Total 26263 (delta 277), reused 320 (delta 181), pack-reused 25768        
Receiving objects: 100% (26263/26263), 14.68 MiB | 3.29 MiB/s, done.
Resolving deltas: 100% (17147/17147), done.
Cloning into '/Users/krzyh/code/clonetest/saleor-storefront'...
remote: Enumerating objects: 50, done.        
remote: Counting objects: 100% (50/50), done.        
remote: Compressing objects: 100% (28/28), done.        
remote: Total 30401 (delta 18), reused 43 (delta 17), pack-reused 30351        
Receiving objects: 100% (30401/30401), 15.11 MiB | 1.84 MiB/s, done.
Resolving deltas: 100% (20493/20493), done.
Submodule path 'saleor': checked out 'f354d5b80f0924dac6c8b7d5851995ddbfb48a22'
Submodule path 'saleor-dashboard': checked out '0d9f8662299a98de4c89bbdf5b8142a0e1790bc7'
Submodule path 'saleor-storefront': checked out '861675df1e50df3d28ec800585c46bbc7edcc6a2'

~/code took 29s 
❯ cd clonetest 

clonetest on  master 
❯ cd saleor

saleor on  HEAD (f354d5b) is 📦 v0.0.0 via ⬢ v12.18.1 via 🐍 v2.7.16 
❯ ls
CHANGELOG.md		app.json		package.json		saleor
Dockerfile		common.env		poetry.lock		scripts
LICENSE			deployment		pyproject.toml		setup.cfg
Procfile		locale			requirements.txt	templates
README.md		manage.py		requirements_dev.txt	tests
SECURITY.md		package-lock.json	runtime.txt		tox.ini
```